app/static/js/admin/guest/explore/event-box-component.js

```diff
@@ -22,7 +22,9 @@ Vue.component('event-box', {
             return moment(this.event.start_time).format('ddd, MMM DD HH:mm A');
         },
         locationName: function () {
-            return this.event.location_name.split(",")[0];
+            if (this.event.location_name) {
```

Comment:
@nikhilrayaprolu also, we need to return a default value ... Since this is a computed value